exporter/jaeger/agent.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package jaeger contains an OpenCensus tracing exporter for Jaeger.
+package jaeger // import "go.opencensus.io/exporter/jaeger"
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	"git.apache.org/thrift.git/lib/go/thrift"
+	gen "go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger"
+)
+
+// UDPPacketMaxLength is the max size of UDP packet we want to send, synced with jaeger-agent
+const UDPPacketMaxLength = 65000
+
+// AgentClientUDP is a UDP client to Jaeger agent that implements gen.Agent interface.
+type AgentClientUDP struct {
```

Comment:
Don't export this, these types should not be visible to the user.

agentClientUDP